test/rules/assertions/checkParamNames.js

```diff
@@ -231,7 +231,7 @@ export default {
            * @param cfg.foo
            * @param cfg.foo
            */
-          function quux ({foo, bar}) {
+          function quux ({foo}) {
```

Comment:
Could you indicate what the reasons were for each of the cases where the tests were modified? That should make reviewing faster. While we normally hopefully won't need to refactor old tests and should just add tests, I understand that in these tests we were previously ignoring destructured properties rather than reporting them. But if you could indicate for each change why the change was needed (whether a removal or even an addition to a preexisting test), that would help (e.g., why were underscores or numbers added to some variable names, etc.). I'll mark the specific lines besides this one.

Reply:
This change removed the `bar` property since the test was for duplicate properties, not non-existent properties. Leaving it as `{foo, bar}` results in the test failing and only reporting the missing property.